daemon/images/image.go

```diff
@@ -24,52 +24,41 @@ func (e ErrImageDoesNotExist) Error() string {
 // NotFound implements the NotFound interface
 func (e ErrImageDoesNotExist) NotFound() {}
 
-// GetImageIDAndOS returns an image ID and operating system corresponding to the image referred to by
-// refOrID.
-// called from list.go foldFilter()
-func (i ImageService) GetImageIDAndOS(refOrID string) (image.ID, string, error) {
+// GetImage returns an image corresponding to the image referred to by refOrID.
+func (i *ImageService) GetImage(refOrID string) (*image.Image, error) {
```

Comment:
Same here, we could consider renaming to `ImageService.Get()`

Reply:
Responding to all the naming comments here.

The `ImageService` (at least for now) is actually responsible for both images and layers. Maybe in the future we can split these, but it would probably be better to wait for containerd snapshotter integration, as that might change things.

I think it might be better to keep the `Image` and `Layer` in the names for these methods until we can split them. I also wanted to keep the diff as small as possible. Renaming will cause a lot more changes. 

There's still a lot more work to clean up these interfaces, so maybe we can address the renames in a follow up?

Reply:
Definitely; I just saw the names and thought: image in the name is a bit redundant. I agree with keeping the diff small, and we make such changes in future